### PR TITLE
Upgrade to the identity-base v2 API

### PR DESCRIPTION
**Pull request by tux3**

### CODE REVIEW

File: tanker_identity/crypto/__init__.py [hunk 1]
```diff
@@ -27,6 +28,13 @@ def sign_keypair():
     return crypto_sign_keypair()
 
 
+def box_keypair():
```
Comment (dmerejkowsky) [inline tanker_identity/crypto/__init__.py]: there seem to be no tests for this function ...

File: tanker_identity/identity/__init__.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    }
+    sig_pub_key, sig_priv_key = tanker_identity.crypto.sign_keypair()
+    signature_keys = {
+        "public_key": base64.b64encode(sig_pub_key).decode(),
+        "private_key": base64.b64encode(sig_priv_key).decode(),
+    }
+    return encryption_keys, signature_keys
+
+
+def generate_user_token(trustchain_id, trustchain_private_key, user_id):
```
Comment (dmerejkowsky) [inline tanker_identity/identity/__init__.py]: there's still a public function called generate_user_token ?

File: tanker_identity/test/test_identity.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    )
+    identity = parse_b64_json(b64_identity)
+    return identity
+
+
+def test_generate_identity_happy(test_trustchain):
+    identity = generate_test_identity(test_trustchain)
+    delegation_signature = base64.b64decode(identity["delegation_signature"])
+
+    assert identity["trustchain_id"] == test_trustchain["id"].decode()
```
Comment (dmerejkowsky) [inline tanker_identity/test/test_identity.py]: not your fault, but the test_trustchain should contain only strings, no bytes

File: tanker_identity/test/test_identity.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    identity_alice = parse_b64_json(tanker_identity.identity.create_provisional_identity(
+        test_trustchain["id"],
+        "alice@gmail.ru"
+    ))
+    identity_bob = parse_b64_json(tanker_identity.identity.create_provisional_identity(
+        test_trustchain["id"],
+        "bob@office360.com"
+    ))
+
+    assert identity_alice["encryption_key_pair"]["public_key"] != identity_bob["encryption_key_pair"]["public_key"]
```
Comment (dmerejkowsky) [inline tanker_identity/test/test_identity.py]: use a loop:

```python
for pair in ["encryption_key_pair", "signature_key_pair"]:
   ...
```

**Comment by dmerejkowsky** [comment]
Also, please use strings in conftests.py, not bytes

File: tanker_identity/test/test_identity.py (hunk 3 above)
Comment (tux3) [reply tanker_identity/test/test_identity.py]: Since you offered to off-list, I'd be happy to let you do that =]

**Pull request merged**